scripts/userdb/mysqlsetup.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env sh
```

Comment:
The politeiawww README needs a example walkthrough of setting up the mysql db using this script.  See the politeiad README as an example.

Add a `Setup MySQL (optional)` section in the README in between the `Install politeia` and the `Setup and run politeiawww` sections.